cmd/local.go

```diff
@@ -236,6 +245,11 @@ func local(cmd *cobra.Command, args []string) {
 		viper.Debug()
 	}
 
+	//Check for parallel mode provided with more nodes than CPU Cores
+	if viper.GetInt("parallel.nodes") > runtime.NumCPU() || viper.GetInt("parallel.nodes")*viper.GetInt("threads") > runtime.NumCPU() {
```

Comment:
I don't think this is valid, as I _believe_ you can run in parallel across multiple nodes. It doesn't happen that often but we should be careful around forcing something to fail

Reply:
Yeah this has already been removed locally. Made sense when I was putting everything together, but not as I thought it through and started trying grid execution